odoo/addons/base/views/res_users_views.xml

```diff
@@ -44,12 +44,15 @@
                 </list>
             </field>
         </record>
-        <record id="change_password_wizard_action" model="ir.actions.act_window">
+        <record id="action_change_password_wizard" model="ir.actions.server">
             <field name="name">Change Password</field>
-            <field name="res_model">change.password.wizard</field>
-            <field name="view_mode">form</field>
-            <field name="target">new</field>
+            <field name="model_id" ref="base.model_res_users"/>
             <field name="binding_model_id" ref="base.model_res_users"/>
+            <field name="state">code</field>
+            <field name="code">
+                action = records.action_change_password_wizard()
+            </field>
```

Comment:
I would avoid to convert this action from a `ir.actions.act_window` to a `ir.actions.server` because it will store in database yet another server action, meaning code saved in db. Is it strictly necessary ? Can't you preserve the window action instead ?